### PR TITLE
[Merged by Bors] - feat(RingTheory/RootsOfUnity/Complex): complex roots of unity have norm 1

### PR DESCRIPTION
**Pull request by MichaelStollBayreuth**
This adds
```lean
lemma Complex.norm_rootOfUnity_eq_one {ζ : ℂˣ} {n : ℕ+} (hζ : ζ ∈ rootsOfUnity n ℂ) :
    ‖(ζ : ℂ)‖ = 1
```

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/RootsOfUnity/Complex.lean [hunk 1]
```diff
@@ -186,3 +187,10 @@ theorem IsPrimitiveRoot.arg {n : ℕ} {ζ : ℂ} (h : IsPrimitiveRoot ζ n) (hn
     exact mod_cast not_le.mp h₂
   · exact Nat.cast_pos.mpr hn.bot_lt
 #align is_primitive_root.arg IsPrimitiveRoot.arg
+
+lemma Complex.norm_rootOfUnity_eq_one {ζ : ℂˣ} {n : ℕ+} (hζ : ζ ∈ rootsOfUnity n ℂ) :
```
Comment (erdOne) [inline Mathlib/RingTheory/RootsOfUnity/Complex.lean:191]:
```suggestion
lemma Complex.norm_eq_one_of_mem_rootOfUnity {ζ : ℂˣ} {n : ℕ+} (hζ : ζ ∈ rootsOfUnity n ℂ) :
```
Or maybe `norm_of_mem_rootOfUnity`?

Comment (MichaelStollBayreuth) [reply Mathlib/RingTheory/RootsOfUnity/Complex.lean]: OK (I'll stick with "_eq_one").

**Comment by erdOne** [comment]
Thanks!
maintainer merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**